### ISSUE TITLE
[Bug] Horses do not make noise when running

### ISSUE THREAD

**Issue by dizzyrune** [sekelsta/horse-colors#78]
The running sounds for vanilla horses are pretty great. Unfortunately, it appears that genetic horses don't use these particular sounds. 
Any chance of this being changed in the future?

https://github.com/sekelsta/horse-colors/assets/151792675/0be083df-10ff-4e7f-8b10-509e60b8cc77

https://github.com/sekelsta/horse-colors/assets/151792675/c1049490-3cac-4036-8922-0e58fb020ce2

**Comment by sekelsta** [comment]
Version please, this used to work

**Comment by dizzyrune** [comment]
At the time of posting, it would have been the latest version on Modrinth, 1.20.1-13.2.

I’ve double-checked and done some tests with only Realistic Horse Genetics installed. 
For 1.20.1 (Forge: 47.3.0, 47.2.0, and NeoForge: 47.1.105), I tried 13.2, 12.6, 12.4, 11.5 via Modrinth and 13.0 via Curseforge.
I also tested:
1.19.2-13.1, 1.17.1-1.7.1, 1.15.2-1.4.0.1 via Modrinth
1.15.2-(1.5.1.2, 1.3.6, 1.2.4, 1.2.3) via Curseforge

Really wanted to be thorough. After 1.15.2-1.2.3 I tried 1.15.2-1.1.3 and can confirm 1.1.3 has running sounds for genetic horses. Unfortunately I don't think any version after that one has the running sounds for genetic horses.

I hope this information helps, your mod is brilliant! I hope it's an easy fix. I'd be very happy to assist if you'd like a hand testing any fixes.
